### DISCUSSION TITLE
Log 메시지 폼에 대하여

### DISCUSSION THREAD

**Discussion by laigasus** [Strong-Potato/TripVote-BE#19] [Polls]
향후 APM 운영시 로그 수집할 때 영향이 클 것 같긴하네요.

https://github.com/Strong-Potato/TripVote-BE/pull/18#discussion_r1441679112 참고

**Comment by Dr-KoKo** [comment]
향후 `ExceptionHandler` 외의 다른 클래스에서 로그를 남기게 되면 하나의 요청에 대해 여러 개 로그가 남게 될 것 같네요. 그 때 같은 요청의 로그인지 구분할 수 있게 대괄호([ ]) 안에 값을 남겨주면 좋을 것 같아요

**Reply by Dr-KoKo** [reply]
대괄호 안에 남길 수 있는 값
1. Session-id : session을 안 쓸 수도 있을 것 같아서 조금 조심스럽네요.
2. Access-token: 로그인하지 않은 유저의 로그는 처리하기 힘들 수도 있겠네요.
3. Principal: 로그인한 경우는 member_id, 로그인하지 않은 경우는 임의의 숫자(ex. 0)를 넣어서 처리할 수 있겠네요. (email을 남기는 것도 가능해보입니다.)
4. 또는 Filter를 하나 만들어서 request의 attribute에 임의의 수(`UUID`)를 부과해주는 것도 방법일 것 같습니다.

**Comment by Dr-KoKo** [comment]
log 레벨로도 충분하다는 쪽이 과반수네요!
